cmd/publicize/server_test.go

```diff
@@ -174,7 +178,42 @@ func TestCheckPrerequisites(t *testing.T) {
 func TestMergeAndPushToRemote(t *testing.T) {
 	publicOrg, publicRepo := "openshift", "test"
 	privateOrg, privateRepo := "openshift-priv", "test"
-
+	fixedTime := time.Now()
+	fixedTimeUnix := fixedTime.Unix()
+	makeRepo := func(localgit *localgit.LocalGit, org, repo string, init func() error) error {
+		if err := localgit.MakeFakeRepo(org, repo); err != nil {
+			return fmt.Errorf("couldn't create fake repo for %s/%s: %v", org, repo, err)
+		}
+		path := filepath.Join(localgit.Dir, org, repo)
+		initial := filepath.Join(path, "initial")
+		if err := os.Chtimes(initial, fixedTime, fixedTime); err != nil {
+			return err
+		}
+		cmd := exec.Command("git", "add", initial)
+		cmd.Dir = path
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+		if err := cmd.Run(); err != nil {
```

Comment:
Nit: To provide more useful error messages, `cmd.CombinedOutput` is very useful

Reply:
I added the output redirection for that, but `CombinedOutput` does look better in this case.  Updated.